conf.yaml

```diff
@@ -56,6 +56,11 @@ targets:
       - aur.archlinux.org
     icon: images/archlinux.png
     twitter: "@archlinux"
+  Ars Technica:
+    href: https://arstechnica.com
+    hosts:
+      - arstechnica.com
```

Comment:
also:
- arstechnica.net
- cdn.arstechnica.net

Reply:
The Domain `arstechnica.net` doesn't resolves neither to an A nor an AAAA record.

```
; <<>> DiG 9.10.6 <<>> arstechnica.net A
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 23241
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;arstechnica.net.		IN	A
```

I'll add `cnd.arstechnica.net`.

Reply:
Yeah, I was just quickly checking uMatrix and it obviously added the domain because the cdn subdomain existed below it.

Reply:
Cool tool. I used IPvFoo to check which domains could be interesting.